Detectors/MUON/MCH/Status/src/HVStatusCreator.cxx

```diff
@@ -17,12 +17,53 @@
 #include "MCHGlobalMapping/Mapper.h"
 #include "MCHStatus/StatusMap.h"
 
+float sum(float s, o2::dcs::DataPointValue v);
+
 namespace o2::mch
 {
 
 void HVStatusCreator::findBadHVs(const DPMAP& dpMap)
 {
+  // decode the DCS DPMAP
+  DPMAP2 dpsMapPerAlias = decodeDPMAP(dpMap);
+  
+  // Find list of HV issues per alias 
+  for (const auto& [alias, dpsHV] : dpsMapPerAlias) {
+    int chamber = o2::mch::dcs::toInt(o2::mch::dcs::aliasToChamber(alias));
+    std::vector<TimeRange> hvIssuesList;
+
+    uint64_t tStart, tStop = 0;
+    bool ongoingIssue = false;
+
+    for (auto& [timestamp, valueHV] : dpsHV) {
+      if (valueHV < StatusMapCreatorParam::Instance().hvLimits[chamber]) { // check whether HV point is below set threshold for chamber
```

Comment:
It would probably be more efficient to get the HV limit for this chamber once outside of the loop over data points